spa/src/components/donationPage/pageContent/DPayment.js

```diff
@@ -60,25 +35,22 @@ function NotLivePlaceholder() {
 }
 
 export function PayFeesWidget() {
-  const { page, frequency, amount, payFee, setPayFee } = usePage();
-
+  const { page, frequency, userAgreesToPayFees, setUserAgreesToPayFees, feeAmount } = usePage();
   const currencySymbol = page?.currency?.symbol;
 
   return (
     <S.PayFees data-testid="pay-fees">
       <S.PayFeesQQ>Agree to pay fees?</S.PayFeesQQ>
       <S.Checkbox
         label={
-          amount
-            ? `${currencySymbol}${formatStringAmountForDisplay(
-                calculateStripeFee(amount, frequency, page.revenue_program_is_nonprofit)
-              )} ${getFrequencyAdverb(frequency)}`
+          feeAmount
+            ? `${currencySymbol}${formatStringAmountForDisplay(feeAmount)} ${getFrequencyAdverb(frequency)}`
             : ''
         }
         toggle
-        checked={payFee}
-        onChange={(_e, { checked }) => setPayFee(checked)}
-        data-testid={`pay-fees-${payFee ? 'checked' : 'not-checked'}`}
+        checked={userAgreesToPayFees}
+        onChange={(_e, { checked }) => setUserAgreesToPayFees(checked)}
```

Comment:
Usually when I have to name a variable that is useless, I just use `_`. I've seen this pattern somewhere and used it since.
```suggestion
        onChange={(_, { checked }) => setUserAgreesToPayFees(checked)}
```

Reply:
Yeah, I do that as well. this was mainly existing code, but i'll update.